ics/event.py

```diff
@@ -523,6 +533,17 @@ def categories(event, line):
             event.categories.update({unescape_string(cat)})
 
 
+@Event._extracts('ATTENDEE')
+def attendee(event, line):
+    if line:
+        # skip the "mailto:" part to get only email
```

Comment:
an `assert line.value[:7] == "mailto:"` might be useful to avoid eating garbage if the input is invalid. 

> fail early, fail often.